www/js/pages/maltainvest-account-opening/maltainvest-account-opening.controller.js

```diff
@@ -182,12 +184,27 @@
             websocketService.sendRequestFor.createMaltainvestAccountSend(params);
         };
 
+        vm.acceptRisk = () => {
+            acceptRisk = 1;
+            vm.submitAccountOpening();
+        };
+
+        vm.declineRisk = () => {
+            const params = _.clone(vm.data);
+            params.accept_risk = 0;
+            websocketService.sendRequestFor.createMaltainvestAccountSend(params);
```

Comment:
@nazaninreihani 
It's better to use `vm.submitAccountOpening` by passing `params` to it. in this way the logics are in one place.